projects/js-packages/components/components/product-price/index.tsx

```diff
@@ -26,9 +27,17 @@ const ProductPrice: React.FC< ProductPriceProps > = ( {
 
 	showNotOffPrice = showNotOffPrice && offPrice != null;
 
+	const offPercentage = 100 - Math.round( ( offPrice / price ) * 100 );
```

Comment:
What do you think about making this something more generic like `label` (instead of `showOffPercentage`? That way we could still use it for things like 50% off but also for stuff like `NEW` or `PROMO`.